app/frontend/src/components/Button/Button.tsx

```diff
@@ -55,6 +56,8 @@ export default function Button<D extends ElementType = "button", P = {}>({
     try {
       setWaiting(true);
       await onClick(event);
+    } catch (e) {
+      Sentry.captureException(e);
```

Comment:
Is this supposed to be here? I thought it should capture exceptions automatically in the error boundary

Reply:
This is needed here since even without the catch, I believe JS catches the error (equivalent of writing an empty catch block) and so it won't go to the error boundary. The error boundary only gets uncaught errors